GetF!t/Modules/Exercises/Timer/TimerView.swift

```diff
@@ -1,7 +1,7 @@
 import SwiftUI
 
 struct TimerView: View {
-    @ObservedObject var timerViewModel: TimerViewModel
+    @StateObject var timerViewModel = TimerViewModel(exerciseName: LocalizedStringProvider.ExercisesNames.squat)
```

Comment:
Toto je riadna prasárna 😄 Asi nie je dobré aby timer view mal vždy title squat 😄  sprav to nejak inak. Buď si ten title drž vo view (tým pádom to predáš v inite) alebo vymysli niečo iné 🙂

Reply:
done